console/calling_commands.rst

```diff
@@ -18,16 +18,15 @@ the returned code from the command (return value from command ``execute()``
 method)::
```

Comment:
This paragraph needs rework, it talks about the `execute()` method which no longer exists in the example.